packages/cli/src/commands/urls.ts

```diff
@@ -44,15 +48,18 @@ export default class Urls extends ProfileCommand<typeof Urls> {
       insecureSkipVerify: flags['insecure-skip-verify'],
     }
 
+    const tunnelingKey = await pStore.getTunnelingKey()
     const { clientId, rootUrl } = await tunnelServerHello({
       tunnelOpts,
       knownServerPublicKeys: pStore.knownServerPublicKeys,
-      tunnelingKey: await pStore.getTunnelingKey(),
+      tunnelingKey,
       log: this.logger,
     })
 
     const flatTunnels = await commands.urls({
       rootUrl,
+      includeAccessCredentials: !!flags['include-access-credentials'],
```

Comment:
`!!` seems redundant (if there's a default)

Reply:
Refactored a bit, include-access-credentials is no longer passed to core/lib.
The reasoning is that is more about manipulation and formatting to the URLs, I don't think internal "up"/"urls" need to take care of that. (It also removes details, since we no longer have the raw URL of the tunnels).

I think that for plugins, we'll probably want to send both (url, and "secure-url") or alternatively, expose convinence functions for embedded basic-auth credentials in a url